variables.scad

```diff
@@ -14,23 +14,26 @@ SensorHole_ledge = 1.0;
 
 wall_thickness = 2;
 
-UsbCover_height = 10;
+UsbCover_height = 9.7;
 UsbCover_foot_height = 1.5;
 UsbCover_width = 15.5;
 UsbCover_depth = 6.5;
 UsbCover_foot_length = 2.5;
 UsbCover_radius = 1.5;
-UsbCover_magnet_size = 3.0; // increase if your printer needs more clearance for press fit
-UsbCover_magnet_depth = 3.3;
+
+UsbCover_r1 = 1.8; // the rounded corners with smaller radius
+UsbCover_r2 = 2; // the rounded corners with bigger radius
+UsbCover_magnet_size = 3.0; // decrease if magnets are too loose
+UsbCover_magnet_clearance = 0.25; // increase if your printer needs more clearance for press fit
+UsbCover_magnet_depth = 3.2;
 UsbCover_magnet_spacing = 6.5; // center to center
 
 UsbCover_indent_depth = 1.5;
-UsbCover_indent_radius = 0.85;
 UsbCover_indent_height = 4;
 UsbCover_indent_width = 12;
 
-UsbCover_wing_offset = 1.88;
-UsbCover_wing_width = 2.75;
+UsbCover_wing_offset = 2.2;
+UsbCover_wing_width = 2.6;
```

Comment:
Why are these values so different from what I measured?

I might need to testprint this.

Reply:
Because we were looking at two different initial designs - I was using the latest version and had to measure on a ``.stl`` which is cumbersome, you went around that part by using the ``.step`` file from an "older" folder.

The easiest way to verify this matches what we currently have is to subtract this model and the latest .stl in openscad - I will add a line to that end.

Reply:
![image](https://user-images.githubusercontent.com/44007906/148612800-c5ca7614-cb49-4920-8d72-88ef7167a5b5.png)
![image](https://user-images.githubusercontent.com/44007906/148612883-bfa16504-14c6-4bae-96c8-2a7f0b1a21d1.png)

compared using the following import lines in UsbCover.scad
```
translate([UsbCover_width+5,0,0])translate([-50.1,UsbCover_depth,0])rotate([0,270,90])import("../../MainCase/VerticalCase_JSN-AJ/OBS-MainCase-A-003_USB-Cover_v0.1.1.stl");
translate([-UsbCover_width-5,0,0])translate([-UsbCover_width/2,0.15,0])import("../../MainCase/VerticalCase_JSN-AJ/OldVersions/OBS-MainCase-A-003_USB-Cover_v0.1.0.stl");
```